docs/README.md

```diff
@@ -0,0 +1,22 @@
+---
+home: true
+title: Documentation | CC Global Components
+lang: en-US
+footer: Copyright (c) 2022 Creative Commons
```

Comment:
Can this footer text use markdown links? Let's change the text as follows and add markdown links to the license if possible.

```suggestion
footer: Except where otherwise [noted](https://creativecommons.org/policies#license), content on this site is licensed under a [Creative Commons Attribution 4.0 International license](https://creativecommons.org/licenses/by/4.0/).
```